<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="progress_bar_set_50">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set 50%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progress_bar_new_range">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progress_bar_range">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="progress_bar_reset">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
